lib/Github/HttpClient/HttpClient.php

```diff
@@ -162,7 +194,7 @@ public function authenticate($tokenOrLogin, $password = null, $method)
     }
 
     /**
-     * @return Request
+     * @return \GuzzleHttp\Message\Request|\Guzzle\Http\Message\Request
```

Comment:
this is painful IMO. As the library provides its own HttpClient interface, it should hide differences inside it IMO (and btw, this signature does not match the interface phpdoc anymore)